docs/triggers/trigger-generic-inputs.md

```diff
@@ -0,0 +1,56 @@
+# Trigger Generics
+
+When creating or updating a trigger at runtime, a Workflow definition can be passed into the `client.workflows.triggers.create` and `client.workflows.triggers.update` methods as a generic which will allow for typeahead/autocomplete of the input parameters based on the definition's inputs. 
+
+## Defining A Workflow
+
+Recall that in the [SDK](https://github.com/slackapi/deno-slack-sdk/blob/main/docs/workflows.md#workflows), a `workflow` can be defined using the `DefineWorkflow` method.
+When a workflow is created in this manner, if contains a `workflow.definition` object. By passing this definition into the `create` method, the trigger will have access to information related to the workflow's definition.
+
+### Using Trigger Generics at runtime Example 
+
+```ts
+import type { SlackFunction } from "/deno-slack-sdk/src/types.ts";
+import { TriggersFunction, TriggersWorkflow } from "../manifest.ts";
+import { SlackAPI } from "../../deno-slack-api/src/mod.ts";
+
+const triggers: SlackFunction<typeof TriggersFunction.definition> = 
```

Comment:
You should follow the lead on this doc on how to fill out function runtime logic: https://api.slack.com/future/functions/custom#implement

Ensure you test it out yourself to make sure it works!

Reply:
@jmplexi you don't need to pass in the generic here anymore. Check the link I linked to above, instead the workflow definition is passed in as the first parameter to `SlackFunction`.